lib/api/core/plugins/pluginContext.js

```diff
@@ -70,23 +69,29 @@ class PluginContext {
       Dsl: 'Koncorde'
     });
 
-    this.errors = require('kuzzle-common-objects').errors;
+    this.errors = errors;
 
     this.secrets = JSON.parse(JSON.stringify(kuzzle.vault.secrets));
 
     if (kuzzle instanceof Kuzzle) {
       const
-        // Lowercasing the plugin name is needed because Elasticsearch
-        // forbids uppercased characters in index names.
-        pluginInternalIndex = `%plugin:${pluginName}`.toLowerCase(),
-        pluginInternalEngine = new InternalEngine(kuzzle, pluginInternalIndex),
+        pluginIndex = `plugin:${pluginName}`.toLowerCase(), // uppercase are forbidden by ES
+        // @todo use internalStorage
+        pluginInternalEngine = {
+          init: () => {},
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this function is empty or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1186)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AEmptyBlock)